storage/connectors/bigquery/src/main/java/feast/storage/connectors/bigquery/writer/FeatureSetSpecToTableSchema.java

```diff
@@ -76,19 +84,42 @@ public void processElement(
       @Element KV<String, FeatureSetProto.FeatureSetSpec> element,
       OutputReceiver<KV<String, TableSchema>> output,
       ProcessContext context) {
-    Schema schema = createSchemaFromSpec(element.getValue(), element.getKey());
-    output.output(KV.of(element.getKey(), serializeSchema(schema)));
+    String specKey = element.getKey();
+
+    Table existingTable = getExistingTable(specKey);
+    Schema schema = createSchemaFromSpec(element.getValue(), specKey, existingTable);
```

Comment:
should building the schema be moved down before `createTable(specKey, schema);`? just for a bit more efficiency